bot/resources/utilities/py_topics.yaml

```
@@ -54,12 +64,13 @@
 342318764227821568:
     - Where do you get your best data?
     - What is your preferred database and for what use?
+    - What are examples of unsafe usage of databases?
```

I'm not sure this one really fits in with all the other questions as it's worded right now, considering just about every other question asks about personal opinions/positions.

Maybe it could be changed to something like this:

```suggestion
    - What is the least safe use of databases you've seen?
```

---

I think this is a better implementation of my question, I wasn't really sure about it in the first place.